packages/serverpod/lib/src/database/database.dart

```diff
@@ -39,10 +39,19 @@ class Database {
   })  : _session = session,
         _databaseConnection = DatabaseConnection(poolManager);
 
-  /// Find a list of [TableRow]s from a table, using the provided [where]
-  /// expression, optionally using [limit], [offset], and [orderBy]. To order by
-  /// multiple columns, user [orderByList]. If [where] is omitted, all rows in
-  /// the table will be returned.
+  /// Returns a list of [$TableRow]s matching the given query parameters.
```

Comment:
Suggestion: There shouldn't be a $ infront of TableRow.

Reply:
my bad, removed that.